packages/spark-core/components/_accordions.scss

```diff
@@ -81,6 +81,7 @@
 
 .sprk-c-Accordion--navigation .sprk-c-Accordion__summary {
   border-top: 0;
+  height: $icon-xl;
   min-height: $icon-xl;
```

Comment:
Hey @janinavelasco9, I'm curious about what the reasoning was for having both a min-height and height on this?

Reply:
IE11 doesn't respect min-height. so i had to add a height for IE11 to center the content.

Reply:
basically min-height on a flex container won't apply to its flex items. so i put the height on there for IE11 to recognize the height. maybe there is a better way to go about it?

Reply:
@janinavelasco9  i think we need that to be removed. @RCopeland  is going to take get with you. I gotta head out for the day.

Reply:
Approving for @RCopeland to take over

Reply:
@janinavelasco9 adding the height wont work for the use case where we add like a sentence to the summary. The text will just overflow its container : (